core/src/test/java/org/jdbi/v3/core/mapper/reflect/ImmutableTest.java

```diff
@@ -71,93 +71,103 @@ private void mockAllNullsResult() throws SQLException {
 
     @Test
     public void shouldSetValueWithBuilder() throws Exception {
-        int aIntVal = 42;
+        int aIntVal1 = 42;
         long aLongVal = 100L;
         String aStringValue = "Foo";
+        int aIntVal2 = 42;
 
-        mockColumns("id", "name", "valueInt");
+        mockColumns("id", "name", "valueInt", "inner_id");
```

Comment:
Just a reminder: these tests still need to be changed into integration tests, and using a real database. We need to see that immutable mappers work as expected with the whole of Jdbi--not just in isolation.